tests/test_aea.py

```diff
@@ -166,7 +166,7 @@ def test_react():
             dummy_handler = dummy_skill.handlers[0]
 
             expected_envelope = envelope
-            agent.outbox.put(expected_envelope)
+            agent._multiplexer.in_queue.put(expected_envelope)
```

Comment:
to make the test succeed, in order to handle an envelope with a not supported protocol, we can't rely on `outbox.put` because that would be filtered out by the multiplexer.

Reply:
Ok so why don't you add the correct list of supported protocols?

Reply:
I should add `"unknown_protocol"` to the list of supported protocols of `OEFLocalConnection` which is not super-neat.
Anyway, with the change to `restricted_to_protocols` this problem will be solved.

Reply:
@marcofavorito can you solve this now?